<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculatrice</class>
 <widget class="QWidget" name="Calculatrice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSpinBox" name="nombre1"/>
   </item>
   <item>
    <widget class="QComboBox" name="operation">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>+</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>*</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>/</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="nombre2"/>
   </item>
   <item>
    <widget class="QPushButton" name="boutonEgal">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultat">
     <property name="text">
      <string>RÃ©sultat</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
